Doc (#101)
dwrtc/dwrtc: docs/websocket.md

* [WebSocketHandler.onReceiveMessageFromWebSocket] uses the session ID to get the [InternalClient]. This is then used to send a message through the P2P layer
* [WebSocketHandler.onReceiveMessageFromSignaling] uses the session ID to get the WebSocket session. This is then used to send the message to the specific WebSocket

### IDs

> The WebSocket session IDs, generated by Javalin, are reused for the user's session ID. The ID is assumed unique. 

Reviewer: ```suggestion
The WebSocket session IDs, are reused for the user's session ID. The ID is assumed to be unique. 
```
